src/client/decorators/SVGDecorator/Core/SVGDecorator.Core.js

```diff
@@ -232,38 +243,37 @@ define([
         }
     };
 
-    SVGDecoratorCore.prototype._updateSVGContent = function (svg) {
-        var svgIcon;
-        //set new content
-        this.$svgContent.empty();
-        this.$svgContent.removeClass();
-        this.$svgContent.addClass('svg-content');
-        if (svg) {
-            this.$svgContent.addClass(svg.replace(REGEXP.INVALID_CSS_CHARS, '__'));
```

Comment:
Make sure to remove this from the docs. I don't think anyone used it..
https://github.com/webgme/webgme/wiki/SVG-Icons